### ISSUE TITLE
Pull Tour Path from Database

### ISSUE THREAD

**Issue by Angelica-P** [SEG491X-Team16/uOttawa-Live-Tour#58]
Once the database is merged in, #17, we need to set up the app so that it pulls the path from the database instead of the hardcoded  version in the app.

As part of this, we also need to consider how to put the tour in the database. Whether we just modify it via the firebase console or make our own thing to that? Or do we just make the assumption that the path is fairly stable and requires few changes in which case just using the console is fine.

**Comment by Angelica-P** [comment]
won't do
